source/reference/limitations.txt

```diff
@@ -53,6 +53,7 @@ General Limitations
   index defined on the same field(s).
 - .. include:: /includes/fact-atlas-require-indexes-limitation.rst
 - ``mongosync`` doesn't sync users or roles.
+- .. include:: /includes/fact-applyOps.rst
```

Comment:
we already have a bullet for this in the limitations section (5th bullet), so I don't think we need a new one

Reply:
Thanks for catching this! I removed the 5th bullet in favor of keeping the includes to ensure consistency across the 2 staged pages

Reply:
np! I actually prefer the original wording we have because it clearly indicates that we don't support it (as opposed to saying "may not replicate") and because it uses "sync" (which I think we prefer to replicate now) . Could we use that sentence for both pages?

Reply:
I changed to "does not sync" (as opposed to "may not sync"). Apologies if I'm missing something but I don't see sync in the original wording (although it is in the new wording).

Reply:
ah my bad! it looks like the new wording still uses 'replicate' ?